fontes/delegua.ts

```diff
@@ -131,13 +133,14 @@ export class Delegua implements DeleguaInterface {
             prompt: '\ndelegua> ',
         });
 
-        leiaLinha.prompt();
+        const isto = this;
```

Comment:
Aqui não seria melhor criar o `isto` no escopo da classe como uma propriedade ? Assim pode usar nas demais referências onde tem o this.

Reply:
Não é o objetivo da PR, então não precisa ser resolvido agora.